Sources/KituraNet/ClientRequest.swift

```diff
@@ -650,4 +650,9 @@ class HTTPClientHandler: ChannelInboundHandler {
             }
          }
      }
+
+     func errorCaught(ctx: ChannelHandlerContext, error: Error) {
+         // No errors to handle, simply close the channel
```

Comment:
@pushkarnk this looks great. You might want to consider logging the errors when you close? But that's up to you.

Reply:
Yep, makes sense to log an error. I've done that. Thanks @weissi !